src/components/Slider/data.js

```diff
@@ -0,0 +1,22 @@
+
+
+export const sliderItems = [
+  {
+    id: 1,
+    img: "https://cdn.shop-apotheke.com/images/A54/286/88/A5428688-p1.jpg",
```

Comment:
It is perfectly okay to use external sources in this project. But in a real job please remember to use internal sources only.